classes/image_types_ostree.bbclass

```diff
@@ -133,6 +134,8 @@ IMAGE_CMD_ostree () {
 	# deploy SOTA credentials
 	if [ -n "${SOTA_PACKED_CREDENTIALS}" ]; then
 		cp ${SOTA_PACKED_CREDENTIALS} var/sota/sota_provisioning_credentials.zip
+		# Device should not be able to push data to treehub
+		zip -d var/sota/sota_provisioning_credentials.zip treehub.json
```

Comment:
What does this do if three is no `treehub.json`? I wonder for example if this gets renamed at some point and we forget to remove the new name. The `zip` program seems to exit with error code `12`, will that abort the whole build?